.github/workflows/manage-flux-releases.yml

```
@@ -0,0 +1,196 @@
+  pull-requests: write
+
+env:
+  ISSUE_TITLE_PREFIX: 'Flux OCI Promotion:'
+  RELEASE_TAG_PREFIX: 'flux-oci-'
+  RELEASE_FILE: 'flux/releasesconfig.json'
+  LABEL: 'flux-release'
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && github.event.action == 'created' && startsWith(github.event.release.tag_name, github.env.RELEASE_TAG_PREFIX)
```

actionlint (1.7.7)

receiver of object dereference "release_tag_prefix" must be type of object but got "string"

(expression)